tests/debug/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+set(DEBUG_TESTS
+    test_xdd_debug_init.sh
+    )
+
+foreach(TEST ${DEBUG_TESTS})
+  configure_file("${TEST}" "${TEST}" COPYONLY)
+  add_test(NAME "${TEST}"
+    COMMAND sh -c "${CMAKE_CURRENT_BINARY_DIR}/${TEST} 2> ${TESTDIR}/logs/${TEST}.log"
```

Comment:
Why use `sh`? The script file should just be made executable

Reply:
No idea, that is what we originally had before lol. Let me try seeing if modifying it works.

Reply:
If the goal is to redirect the output, the redirection can happen in the test instead

Reply:
Again, this is what originally was there. I didn't add that lol... I am testing it out now to see what the point of it was.

Reply:
Honestly, I don't think we even need to have the log stuff at all... I am super confused on what the goal there was. Maybe the idea was do `-x` or something? Like to print out the statements that were running? Who put that `sh -c` in there originally?

Reply:
I just removed the redirection to the log thing completely. See my latest commit comment. I seriously have no idea what the intention was there, so I just got rid of it.